tokio/src/io/util/reader_stream.rs

```diff
@@ -0,0 +1,92 @@
+use crate::io::AsyncRead;
+use crate::stream::Stream;
+use bytes::{Bytes, BytesMut};
+use pin_project_lite::pin_project;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+pin_project! {
+    /// Convert an [`AsyncRead`](crate::io::AsyncRead) implementor into a
+    /// [`Stream`](crate::stream::Stream) of Result<[`Bytes`](bytes::Bytes), io::Error>. After first error it will
+    /// stop.
+    ///
+    /// This type can be created using the [`reader_stream`](crate::io::reader_stream) function
```

Comment:
Can you move the links out of the text like this?
```md
Convert an [`AsyncRead`] implementor into a ...

[`AsyncRead`]: crate::io::AsyncRead
```
Additionally, can you wrap the text at 80 characters?

Reply:
I moved link locations to comment footer and verified that there are no broken links.

Reply:
It seems like you forgot to move the links in this doc-comment?

Reply:
Thanks. Now both places are good